samples/excel/89-preview-apis/chart-axis.yaml

```diff
@@ -0,0 +1,246 @@
+order: 1
+id: excel-89-preview-apis-chart-axis-
+name: Chart axis
+description: 'Get, set, and remove axis unit, label and title in a chart.'
+host: EXCEL
+api_set:
+    ExcelAPI: 1.7
+script:
+    content: "$(\"#setup\").click(() => tryCatch(setup));
+    $(\"#get-axis-unit\").click(() => tryCatch(getAxisUnit));
+    $(\"#change-axis-unit\").click(() => tryCatch(changeAxisUnit));
+    $(\"#remove-axis-label\").click(() => tryCatch(removeAxisLabel));
+    $(\"#show-axis-label\").click(() => tryCatch(showAxisLabel));
+    $(\"#set-axis-title\").click(() => tryCatch(setAxisTitle));
+    
+    async function getAxisUnit() {
+        await Excel.run(async (context) => {
+            const sheet = context.workbook.worksheets.getItem(\"Sample\");
+    
+            let chart = sheet.charts.getItemAt(0);
+            let categoryAxis = chart.axes.categoryAxis;
+            let valueAxis = chart.axes.valueAxis;
+    
+            categoryAxis.load();
+            valueAxis.load();\_
```

Comment:
`.load()`: same as above.  Also, where are you even using `valueAxis`?

Reply:
valueAxis.displayUnit -->  I changed unit from thousands to hundreds